src/API/slot.js

```diff
@@ -11,15 +11,21 @@ class SlotDataService {
         return http.simpleGet(`/queues/${queue}/slots/${id}`)
     }
 
-    reserve(queue, id, data) {
-        return http.post(`/queues/${queue}/slots/${id}/reserve?next=cancel`, data).catch((err) => {
-            const statusCode = err.response?.status
-            if (statusCode === 403) {
-                errorAlert(err.response.data.description)
-            } else {
-                err.handleGlobally()
-            }
-        })
+    reserve(queue, id, token) {
```

Comment:
Токен - это не обязательное поле. Авторизованные пользователи могут пользоваться этим эндпоинтом просто с помощью кук.

Reply:
Надо еще дефолтное значение прописать, иначе неудобно будет потом пользоваться
```suggestion
    reserve(queue, id, token = None) {
```